### ISSUE TITLE
[0.9.0] `htmllint()` function doesn't default export

### ISSUE THREAD

**Issue by regseb** [linthtml/linthtml#471]
Version 0.9.0 no longer exports the `htmllint()` function by default. I think the problem is with the TypeScript transpiler or the module bundler, because [index.ts](https://github.com/linthtml/linthtml/blob/develop/packages/linthtml/lib/index.ts#L151-L153) exports function by default.

---

```JavaScript
import linthtml from "@linthtml/linthtml";

console.log(linthtml);
```

With _0.9.0_:

```
{
  config_from_path: [Getter],
  find_local_config: [Getter],
  default: [Function: linthtml] {
    fromConfig: [Function: fromConfig],
    create_linters_for_files: [Function (anonymous)],
    from_config_path: [Function (anonymous)],
    Linter: [class Linter],
    LegacyLinter: [class Linter],
    rules: [Array],
    presets: {
      none: [Object],
      accessibility: [Object],
      validate: [Object],
      default: [Object]
    }
  }
}
```

With _0.8.6_:

```
[Function: linthtml] {
  fromConfig: [Function (anonymous)],
  create_linters_for_files: [Function (anonymous)],
  from_config_path: [Function (anonymous)],
  Linter: [class Linter],
  LegacyLinter: [class Linter],
  rules: [Array],
  messages: [Object]
}
```

**Comment by KamiKillertO** [comment]
@regseb Thanks for reporting, I'll look into it.

**Comment by HOKComics** [comment]
Seems like this breaks the LintHTML VSC extension. Celan install of both resluts in the error message "linthtml: lintHTML is not a function. Check your config file" when I open a html file in VSC.

**Comment by KamiKillertO** [comment]
I've started invetigating the problem.
I'm curious @regseb, why is that an issue for you? Are you directly using linthtml somewhere?

**Comment by regseb** [comment]
I am developing a project that aggregates several linters (using at API) : [Metalint](https://github.com/regseb/metalint). I integrate LintHTML and I found this difference by upgrading to version _0.9.0_. I can work around it, but I reported this problem because I don't think this change is voluntary.

**Comment by KamiKillertO** [comment]
I'm happy to hear of people using LintHTML to build other things.
Yeah, it's an unwanted change. The Typescript migration was a challenge, and it looks like I missed a few things.

**Comment by regseb** [comment]
If the default export isn't possible, the `linthtml` method should be exported normally.

| Version | Import |
| --- | --- |
| 0.8.x | `import linthtml from "@linthtml/linthtml";` |
| 0.9.x | `import LintHTML from "@linthtml/linthtml"; const linthtml = LintHTML.default;` |
| 0.10.x | `import { linthtml } from "@linthtml/linthtml";` |

**Comment by KamiKillertO** [comment]
@regseb  I need to think about it. Changing the export once again has many impacts, I'll need to update the vscode extension once again to support 3 different kind of export 😓 . I'm not sure it's good idea

**Comment by regseb** [comment]
You can keep the default export to not break the API. And add the _normal_ export of the `linthtml` method.

```JavaScript
// In 0.9.x and 0.10.x
import LintHTML from "@linthtml/linthtml";
const linthtml = LintHTML.default;

// In 0.10.x
import { linthtml } from "@linthtml/linthtml";
```
